pkg/cloudprovider/aws/fleet/capacity.go

```diff
@@ -77,9 +77,14 @@ func (c *Capacity) Create(ctx context.Context, constraints *cloudprovider.Capaci
 		}},
 	})
 	if err != nil {
-		return nil, fmt.Errorf("creating fleet, %w", err)
+		return nil, fmt.Errorf("creating fleet %w", err)
+	}
+	if len(createFleetOutput.Errors) > 0 {
+		return nil, fmt.Errorf("createFleetErrors %v", createFleetOutput.Errors)
+	}
+	if len(createFleetOutput.Instances) == 0 {
+		return nil, fmt.Errorf("no instance info in the createFleetOutput")
```

Comment:
try to avoid error messages that refer to variable names (i.e. createFleetOutput). Instead, try to make them as plain english as possible w.r.t the concepts. In this example, I'd probably do:

"create fleet returned 0 instances"

Keep in mind, our error style is to wrap and contextualize the message with a bunch of commas, so in context, this will look something like

"failed to reconcile, while allocating capacity, create fleet returned 0 instances"